src/Builder/CommonBuilder.php

```diff
@@ -109,47 +119,66 @@ protected function createBasicTags(): array
 
         $tags = [];
         foreach ($names as $name) {
-            $tags[$name] = (new Behavior\Tag($name, Behavior\Tag::ALLOW_CHILDREN))
-                ->addAttrs(...$this->globalAttrs);
+            $behavior = new Behavior\Tag($name, Behavior\Tag::ALLOW_CHILDREN);
+            $behavior = call_user_func_array([$behavior, 'addAttrs'], $this->globalAttrs);
+            $tags[$name] = $behavior;
         }
-        $tags['a']->addAttrs(
-            $this->hrefAttr,
-            ...$this->createAttrs('hreflang', 'rel', 'referrerpolicy', 'target', 'type')
-        );
-        $tags['br'] = (new Behavior\Tag('br'))->addAttrs(...$this->globalAttrs);
-        $tags['hr'] = (new Behavior\Tag('hr'))->addAttrs(...$this->globalAttrs);
-        $tags['label']->addAttrs(...$this->createAttrs('for'));
-        $tags['td']->addAttrs(...$this->createAttrs('colspan', 'rowspan', 'scope'));
-        $tags['th']->addAttrs(...$this->createAttrs('colspan', 'rowspan', 'scope'));
+
+        $tags['a']->addAttrs($this->hrefAttr);
+        $tags['a'] = call_user_func_array([$tags['a'], 'addAttrs'], $this->createAttrs('hreflang', 'rel', 'referrerpolicy', 'target', 'type'));
+
+        $tags['br'] = new Behavior\Tag('br');
```

Comment:
In general I'd prefer
```
$tags['br'] = (new Behavior\Tag('br'))-> addAttrs($this->globalAttrs);
```
* using `public function addAttrs(array $attrs) {}`
* reduces necessity to use `call_user_func_array` for every assignment when declaring the `Behavior`